### ISSUE TITLE
Can't seem to be able to root my chromebook android subsystem whatever I try.

### ISSUE THREAD

**Issue by charuru4510471** [nolirium/aroc#35]
I have been trying to root my chromebook android subsystem for the last 4 days but really can't seem to figure it out so i have decided to ask you guys for help.

**Comment by CerisWhite** [comment]
This project has long since been quiet, but recently I just made a breakthrough of my own. Feel free to try it out and let me know how it works for you: https://github.com/CerisWhite/CrAnberry

**Comment by Alomor** [comment]
> This project has long since been quiet, but recently I just made a breakthrough of my own. Feel free to try it out and let me know how it works for you: https://github.com/RennBurris/CrAnberry

I tried nolirium's method as well as your method and both of them are not working on Chrome OS 89 on my Lenovo duet. I followed your step-by-step guide and rebooted. Now when I click on an android app to open it, a white circle appears on the app icon and it spins forever, like the app is loading but it never start. I had to revert.

**Comment by ethanaobrien** [comment]
@CerisWhite Yes, that works. I did some tests with this script and with the script you made and your script mounted the android file system as RW to chrome os but not to android - Which kinda defeats the purpose. I did some tests with this this script (The one that doesnt work) and I found that installing SuperSu is what breaks android. I assume that it is because android 9 uses a different boot method - we would need to run @CerisWhite 's script to root the fs, then, step by step, install supersu and find what breaks the system.

**Comment by CerisWhite** [comment]
@ethanaobrien My script does not mount the system RW in it's current state, neither to android or ChromeOS. Mounting system.raw.img read-write on boot causes the image to not function at all. I also very specifically stated in my readme that SuperSU "breaks the image", which is why my script doesn't install it.
